CHANGELOG.md

```diff
@@ -78,6 +78,8 @@
 - Added support for Kong 3.0 upstream `query_arg` and `uri_capture` hash
   configuration to KongIngress.
   [#2822](https://github.com/Kong/kubernetes-ingress-controller/issues/2822)
+- Added support for Gateway API's GatewayClass v1beta1.
+  [#2889](https://github.com/Kong/kubernetes-ingress-controller/issues/2889)
```

Comment:
Probably worth expanding on this:

```suggestion
- Added support for Gateway API's v1beta1 versions of: GatewayClass, Gateway
  and HTTPRoute.
  [#2889](https://github.com/Kong/kubernetes-ingress-controller/issues/2889)
```

Reply:
I made this comment slightly before I realized all 3 APIs were going to get their own PR: let's just double tap that we link each PR on this entry.

Reply:
I just thought that I'll add more links to this very point here when I'll submit the corresponding PRs.